### PR TITLE
Pass serialization context to GraphQL

### PR DESCRIPTION
**Pull request by meyerbaptiste**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

Use case: take into account serialization groups for example.

### CODE REVIEW

File: src/Bridge/Graphql/Resolver/ItemResolverFactory.php [hunk 1]
```diff
@@ -81,7 +84,13 @@ public function createItemResolver(string $resourceClass, string $rootClass): ca
                 $item = $this->itemDataProvider->getItem($resourceClass, \count($identifiers) > 1 ? \implode(';', $identifiers) : $uniqueIdentifier[0]);
             }
 
-            return $item ? $this->normalizer->normalize($item, null, ['graphql' => true]) : null;
+            return $item ? $this->normalizer->normalize(
+                $item,
+                null,
+                ['graphql' => true] + $this->resourceMetadataFactory
+                    ->create($resourceClass)
```
Comment (soyuka) [inline src/Bridge/Graphql/Resolver/ItemResolverFactory.php]: Can't this be `null` if metadata hasn't been found?

Comment (meyerbaptiste) [reply src/Bridge/Graphql/Resolver/ItemResolverFactory.php]: Why? Should not the `ResourceClassNotFoundException` be thrown instead?

Comment (soyuka) [reply src/Bridge/Graphql/Resolver/ItemResolverFactory.php]: Monday :sleeping:

**Comment by dunglas** [comment]
Thanks @meyerbaptiste

**Pull request merged**